libs/tup-components/src/mfa/MfaSmsPanel.tsx

```diff
@@ -7,36 +7,46 @@ import { TicketCreateModal } from '../tickets';
 const MfaSmsPanel: React.FC = () => {
   const smsMutation = useMfaPairSms();
   const [phoneNumber, setPhoneNumber] = useState<string>('');
+  const [hasAttemptedSMS, setHasAttemptedSMS] = useState<boolean>(false);
   const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
+    setHasAttemptedSMS(true);
     smsMutation.mutate({ phoneNumber });
   };
   return (
-    <form
-      onSubmit={(e) => onSubmit(e)}
-      className={`${styles['mfa-form']} s-form`}
-    >
-      <div>
-        <label htmlFor="mfa-phone-number">Enter your Phone Number:</label>
-        <input
-          required
-          id="mfa-phone-number"
-          onChange={(e) => setPhoneNumber(e.target.value)}
-        />
-        {smsMutation.isError && (
-          <InlineMessage type="error" tagName="small">
-            Unable to pair via SMS. If this error persists,{' '}
-            <TicketCreateModal display="link">
-              submit a ticket
-            </TicketCreateModal>
-            .
-          </InlineMessage>
-        )}
-      </div>
-      <Button type="primary" attr="submit" isLoading={smsMutation.isLoading}>
-        Send Token
-      </Button>
-    </form>
+    <>
+      <form
+        onSubmit={(e) => onSubmit(e)}
+        className={`${styles['mfa-form']} s-form`}
+      >
+        <div>
+          <label htmlFor="mfa-phone-number">Enter your Phone Number:</label>
+          <input
+            required
+            id="mfa-phone-number"
+            onChange={(e) => setPhoneNumber(e.target.value)}
+          />
+          {smsMutation.isError && (
+            <InlineMessage type="error" tagName="small">
+              Unable to pair via SMS. If this error persists,{' '}
+              <TicketCreateModal display="link">
+                submit a ticket
+              </TicketCreateModal>
+              .
+            </InlineMessage>
+          )}
+        </div>
+        <Button type="primary" attr="submit" isLoading={smsMutation.isLoading}>
+          Send Token
+        </Button>
+      </form>
+      {hasAttemptedSMS && (
```

Comment:
```suggestion
      {(smsMutation.data || smsMutation.isLoading) && (
```
We don't need a new piece of state here, since `smsMutation` keeps track of whether it's been touched.